hack/assets/providercustomizations.go

```diff
@@ -100,3 +110,35 @@ func gcpCustomizations(obj *unstructured.Unstructured) {
 		}
 	}
 }
+
+func removeClusterDefaultingWebhooks(obj *unstructured.Unstructured) {
+	mutatingWebhookConfiguration := &admissionregistrationv1.MutatingWebhookConfiguration{}
+	if err := scheme.Convert(obj, mutatingWebhookConfiguration, nil); err != nil {
+		panic(err)
+	}
+
+	webhooks := []admissionregistrationv1.MutatingWebhook{}
+	for _, webhook := range mutatingWebhookConfiguration.Webhooks {
+		switch webhook.Name {
+		case "default.cluster.cluster.x-k8s.io",
```

Comment:
can we make this list more dynamic as more providers will be added in the future?